src/components/svg/LinkSvg.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ISvgProps, Svg } from './Svg';
+import * as classname from 'classnames';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+
+export interface ILinkSvgProps extends React.ClassAttributes<LinkSvg> {
+  url: string;
+  target?: string;
+  linkClasses?: string[];
+  svg?: ISvgProps;
+  tooltip?: ITooltipProps;
```

Comment:
This is optional but always in the render? It works fine without any props?

Reply:
I modified the tooltip 1 month ago to render the children in a span if the prop title is empty or undefined to remove the ternary every time we want a tooltips or not to wrap the element